buildlib/azure-pipelines-pr.yml

```diff
@@ -219,3 +219,8 @@ stages:
     dependsOn: [Codestyle]
     jobs:
     - template: io_demo/io-demo.yml
+
+  - stage: jucx_test
+    dependsOn: [Codestyle]
+    jobs:
+    - template: jucx/jucx-test.yml
```

Comment:
on which machine will it run? will it run on RDMA network?
need to add it to test.yml so it would run on full matrix